src/js/main.js

```diff
@@ -286,6 +34,13 @@ function testFinished() { currentTest.done(); }
 function expectEquals() { currentTest.expectEquals.apply(currentTest,
                                                          arguments); }
 
+function _createSuite(name, output) {
+  var element = document.createElement('testrtc-suite');
+  element.name = name;
+  output.appendChild(element);
```

Comment:
You need to update https://github.com/webrtc/testrtc/blob/polymer1/src/js/main.js#L106 and https://github.com/webrtc/testrtc/blob/polymer1/src/js/main.js#L107  to "output" rather than "output_".